### ISSUE TITLE
Nav block list view - Page List (parent) doesn't provide context that it is locked whereas children do. 

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#48018]
The Page List (parent) doesn't provide context that it is locked whereas children do. This is confusing for assistive tech.

_Originally posted by @getdave in https://github.com/WordPress/gutenberg/issues/46939#issuecomment-1427668766_

**Comment by alexstine** [comment]
@getdave or @draganescu Any advice on where to look for adjusting this? I am not sure why page list itself would not identify itself as locked. I think this is a block attribute but wouldn't a locked state get outputted at a fairly global level?

Thanks.

**Comment by getdave** [comment]
@alexstine The lock status of the block for the Navigation block list view (only) is accessed here

https://github.com/WordPress/gutenberg/blob/88e340668962283e0d6afb93db8172f176ad6ff4/packages/block-editor/src/components/off-canvas-editor/block.js#L61

This uses the `useBlockLock` hook from the `@wordpress/block-editor` package. 

I hope that helps?

**Comment by alexstine** [comment]
I am trying to figure out if the issue is with `useBlockLock` or the block itself. I think it might be the page-list block itself. It seems to not have any `templateLock` attribute. I do not know how to update blocks at the moment. According to the hook, it does get the `rootClientId` so I would think it should be able to detect if the parent block is locked.

Figured I would try to take a quick look at this to see if I could make the release but I think this is currently a bit too much out of scope of my current knowledge. I have never studied the block API.

Thanks.

**Comment by draganescu** [comment]
#48723 may remove this issue, or part of it.
